cloudevents/adapters/github.md

```diff
@@ -670,3 +1030,47 @@ based on the specified event.
 | `time`                | Current time                          |
 | `data`                | Content of HTTP request body          |
 
+### WorkflowDispatchEvent
+
+| CloudEvents Attribute | Value                                 |
+| :-------------------- | :------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value |
+| `source`              | "repository.url" value                |
+| `specversion`         | `1.0`                                 |
+| `type`                | `com.github.workflow_dispatch`        |
+| `datacontentencoding` | Omit                                  |
+| `datacontenttype`     | `application/json`                    |
+| `dataschema`          | Omit                                  |
+| `subject`             | "workflow" value                      |
+| `time`                | Current time                          |
+| `data`                | Content of HTTP request body          |
+
+### WorkflowJobEvent
+
+| CloudEvents Attribute | Value                                       |
+| :-------------------- | :------------------------------------------ |
+| `id`                  | "X-GitHub-Delivery" HTTP header value       |
+| `source`              | "repository.url" value                      |
+| `specversion`         | `1.0`                                       |
+| `type`                | `com.github.workflow_job.` + "action" value |
+| `datacontentencoding` | Omit                                        |
+| `datacontenttype`     | `application/json`                          |
+| `dataschema`          | Omit                                        |
+| `subject`             | "workflow_job.name" value                   |
```

Comment:
I  wonder if workflow_job.id is better in the case of a rename?

Reply:
Most of GH ui only show the name, so my assumption is name would be more useful.

I do not know if the id change or not on rename

Reply:
what about workflow_job.run_url?